.gitignore

```diff
@@ -4,6 +4,6 @@ __pycache__/
 *.pyc
 .pycharmrc
 celery.sh
-
+WAM_APP_*/
```

Comment:
What's this for?

Reply:
To leave out the WAM apps out of the version control